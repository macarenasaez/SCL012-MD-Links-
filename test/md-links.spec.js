const mdLinks = require('../src/md-links');


describe('mdLinks', () => {

  it('deberÃ­a traer archivos md desde un directorio', () => {
    return expect(mdLinks.mdLinks('/home/laboratoriad300/Escritorio/SCL012-MD-Links-/src')).resolves.toMatchSnapshot();
  });

});
