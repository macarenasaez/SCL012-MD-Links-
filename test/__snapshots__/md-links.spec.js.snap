// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mdLinks deber√≠a traer archivos md desde un directorio 1`] = `
Array [
  "/home/laboratoriad300/Escritorio/SCL012-MD-Links-/src/README.md",
]
`;
